src/sidebar/components/ModerationBanner.tsx

```diff
@@ -102,9 +102,10 @@ function ModerationBanner({
         {annotation.hidden && <span>Hidden from users</span>}
       </div>
       <div className="self-center pr-2">
-        <ButtonBase
+        <Button
```

Comment:
The theming of this button and the one in `PaginationNavigation` are quite similar. Might be able to consolidate and extract a variant at some point.